lib/src/views/feed/feedroom_screen.dart

```diff
@@ -385,7 +386,27 @@ class _FeedRoomViewState extends State<FeedRoomView> {
   ValueNotifier<bool> rebuildPostWidget = ValueNotifier(false);
   final ValueNotifier postUploading = ValueNotifier(false);
 
-  List<File> imageFiles = [];
+  List<Map<String, dynamic>> imageFiles = [];
+
+  Widget getLoaderThumbnail() {
+    if (imageFiles[0]['mediaType'] == 1) {
```

Comment:
This will result in future bugs, so prefer not using maps like mentioned in prev comment